tests/cli/lib/lib.sh

```diff
@@ -19,12 +19,22 @@ if [ "$COMPOSER_TEST_FAIL_FAST" == "1" ]; then
   }
 fi
 
+
+if [ `rlGetArch` == "ppc64le" ]; then
+    # because there is only qemu-system-ppc64 !
+    ln -s /usr/bin/qemu-system-ppc64 /usr/bin/qemu-system-ppc64le
```

Comment:
I'd set the `QEMU_BIN` variable here, instead of changing the underlying system.